docs/guide/using-python.md

```diff
@@ -10,7 +10,7 @@ A complete description of these functions can be found in the reference guide wh
 
 The Python API of Webots supports both Python 3.7 and Python 2.7.
 On macOS it also supports Python 3.8.
```

Comment:
```suggestion
The Python API of Webots supports Python versions 2.7, 3.7 and 3.8.
Please note that Python 2.7 is deprecated since January 1st, 2020 and should be avoided.
```